pywps/app/Process.py

```diff
@@ -7,7 +7,6 @@
 import os
```

Comment:
Note, this import
https://github.com/geopython/pywps/blob/bde355145462d0767f359e4a8a6303d0606e618d/pywps/app/Process.py#L23

`from pywps._compat import PY2` this doesnt  seem to be necessary any more

Reply:
done